applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -457,7 +467,16 @@ int ble_vcs_discover(struct bt_conn *conn, uint8_t channel_num)
 int ble_mcs_discover(struct bt_conn *conn)
 {
 #if (CONFIG_BT_MCC)
-	return bt_mcc_discover_mcs(conn, true);
+	int ret;
+
+	ret = bt_mcc_discover_mcs(conn, true);
+
+	if (ret) {
+		return ret;
+	}
+
+	mcs_discover_called = true;
```

Comment:
Should the value be set before the call?
If the call fails, the value will not be set.  Will there still be a callback in that case?

Reply:
Then this one is good as is.

Reply:
Shouldn't be a callback if the call fails